hphp/runtime/vm/jit/abi-ppc64.h

```diff
@@ -116,6 +116,10 @@ constexpr RegXMM rFasm = ppc64_asm::reg::f15;
 
 ///////////////////////////////////////////////////////////////////////////////
 
+inline uint8_t required_flags(jit::ConditionCode cc) { return 0xff; }
```

Comment:
The PPC64 (and X64) implementations of this function just set the flags uint8_t to all 1's. The value set for the flags doesn't matter unless the map flags pass is used in either of those two platforms.